apps/artist/client/cta.coffee

```diff
@@ -16,8 +16,8 @@ module.exports = (artist) ->
   # show cta bar
   if !CURRENT_USER and !REFERRER?.includes(APP_URL)
```

Comment:
I'm not sure if this is what @briansw means. This is the code for the older black bar CTA on the artist page.

I think he was referring to updating copy in the [new component](https://github.com/artsy/force/tree/master/components/artist_page_cta).